registration-processor/core-processor/registration-processor-biometric-extraction-stage/src/main/java/io/mosip/registration/processor/stages/biometric/extraction/stage/BiometricExtractionStage.java

```diff
@@ -361,8 +358,8 @@ private IdResponseDTO addBiometricExtractiontoIdRepository(ExtractorDto dto,
 		}
 		List<String> segments=List.of(registrationId);
 		IdResponseDTO response= (IdResponseDTO) registrationProcessorRestClientService.putApi(ApiName.IDREPOEXTRACTBIOMETRICS, segments, extractionFormat, dto.getAttributeName(), null, IdResponseDTO.class, null);
-
 		if (response.getErrors() != null && !response.getErrors().isEmpty()) {
+			idrepoDraftService.idrepodiscardDraft(registrationId);

```

Comment:
In case of reprocessable exception, if we delete the draft, will it not create any issues when reprocessor picks this up ? Was this scenario tested ?

Reply:
According to the discussion. we are not deleting the draft if we get the reprocessable  exception.